src/index.html

```diff
@@ -1,14 +1,34 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header-logo" href="#"><img src="images/logo.png" alt="moyo logo"></a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-link"><a class="is-active" href="#apple">Apple</a></li>
+          <li class="nav-link"><a href="#samsung">Samsung</a></li>
```

Comment:
Checklist:
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/a6fb6311-0085-4654-a7f0-bbc34a525dd1)